app/setup.py

```diff
@@ -342,6 +345,15 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_task_client(application):
+    if application.config["EQ_SUBMISSION_CONFIRMATION_BACKEND"] == "cloud-tasks":
+        application.eq["task-client"] = CloudTaskPublisher(
+            application.config["EQ_SUBMISSION_CONFIRMATION_QUEUE"]
+        )
+    else:
+        application.eq["task-client"] = LogCloudTaskPublisher()
```

Comment:
I think we need to consider the naming. `EQ_SUBMISSION_CONFIRMATION_BACKEND` implies it's specific to submission confirmation, but the assignment is to a generic `task-client`.

Reply:
Makes sense. Only feedback backend then stands out against the rest. But since you can choose your "queue" at the time of the action, it makes sense to be generic.

Reply:
As discussed I've renamed this but have also moved the task creation to the `_publish_fulfilment_request` method